### PR TITLE
Nano time  clock gettime  clock monotonic

### PR DESCRIPTION
**Pull request by michaellilltokiwa**
changes the implementation in c backend of intrinsic fuzion.std.nano_time since clock delivered unsatisfying results.
this test example now seems to return 250ms always.
```
testNanoSleep : time.durations is

  for
    i in 1..20
    last := time.nano.read, now
    _ := time.nano.sleep (ms 250)
    now := time.nano.read
    d := time.duration now-last
  do
    say "$d"
```

### CODE REVIEW

File: src/dev/flang/be/c/Intrinsics.java [hunk 1]
```diff
@@ -417,11 +417,22 @@ CStmnt code(C c, int cl)
         }
       case "fuzion.std.nano_time":
         {
-          return CExpr.call("clock", new List<>())
-            .sub(c._names.GLOBAL_CLOCK_OFFSET)
-            .mul(CExpr.uint64const(1_000_000_000))
-            .div(CExpr.ident("CLOCKS_PER_SEC"))
-            .ret();
+          var result = new CIdent("result");
+          var onFailure = CExpr.call("exit", new List<>(new CIdent("1")){});
```
Comment (fridis) [inline src/dev/flang/be/c/Intrinsics.java:421]: please print a message to stderr and say what happened.

**Review by fridis — COMMENTED** [review]
Looks good. Just minor error-handling improvement requrests.

**Comment by fridis** [comment]
BTW, does this work on windows?

**Comment by michaellilltokiwa** [comment]
> BTW, does this work on windows?

For some reason it is not as stable as on linux:

windows:
image
linux:
image

**Pull request merged**